pycqed/instrument_drivers/physical_instruments/QuTech_ControlBox_v3.py

```diff
@@ -38,6 +38,15 @@ def add_params(self):
                            get_cmd=self._do_get_trigger_source,
                            vals=vals.Enum('internal', 'external',
                                           'mixed'))
+        self._pulse_queue_state = defHeaders.pulse_queue_states[0]
+        self.add_parameter('pulse_queue_state',
+                           get_cmd=self._do_get_pulse_queue_state)
+        self._max_instruction_address = 0
+        self.add_parameter('max_instruction_address',
```

Comment:
I understand what this means because you explained it to me. Again a dosctring would be good.